lib/models/recs.js

```diff
@@ -0,0 +1,154 @@
+const User = require('../models/User');
+const fetch = require('node-fetch');
+
+function calculateTopThree(dataArray1, dataArray2) {
+  const dataArray1Reduced = dataArray1.reduce(function (array, element) {
+    return array.concat(element);
+  });
```

Comment:
What do you try to do here?

Reply:
the data arrays this function receives have nested arrays inside.  I am using concat to turn them all into one normal array